nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -291,6 +293,21 @@ private void setUnconditionalArgumentNullness(
     }
   }
 
+  /**
+   * Generate a StreamNullabilityPropagator handler based on all the stream specifications loaded
+   * from any of our library models.
+   *
+   * <p>This handler must be registered in Handlers.java separatedly from the LibraryModelsHandler
+   * itself.
+   */
+  public StreamNullabilityPropagator getStreamNullabilityPropagatorFromModels() {
```

Comment:
It seems a bit awkward to put this method here as opposed to `StreamNullabilityPropagatorFactory`.  Is it here because it needs access to the `libraryModels` object which is encapsulated within this class?

Reply:
Yeah, `libraryModels` combines all loaded library models from auto-service, and I am not sure I want to provide a way to retrieve it from this class. What we could do is:

1. Add `getCustomStreamNullabilitySpecs()` to this handler, which returns `libraryModels.customStreamNullabilitySpecs()`
2. Add a static method to `StreamNullabilityPropagatorFactory` which takes that and returns the `StreamNullabilityPropagator`

Reply:
I'm fine with whatever you think is cleanest